reference/compiling_for_android.rst

```diff
@@ -16,93 +16,81 @@ Also, you still need to do all the steps mentioned in the
 :ref:`doc_exporting_for_android` tutorial before attempting your custom
 export template.
 
-Requirements
-------------
+A. Requirements
```

Comment:
Well it kind of defeats the purpose of using Sphinx if we can't rely on it to generate the table of contents, but have to do it ourselves by appending "I. A. 1. a. α." to all headings based on their level. If we do want such section identifiers (I'm not sure of the gain for online docs with mostly short pages), I'm pretty sure there's a sphinx configuration option to get them generated automatically. Doing them manually is no-go.